packages/back-end/src/models/SdkConnectionModel.ts

```diff
@@ -250,6 +252,14 @@ export async function editSDKConnection(
     }
   );
 
+  const payloadKey: SDKPayloadKey = {
+    environment: connection.environment,
+    project: connection.project,
+  };
+
+  // Purge CDN if used
+  await purgeCDNCache(connection.organization, [payloadKey]);
```

Comment:
We should only purge if something meaningful has changed.  This should follow the same logic as `needsProxyUpdate`.  For example, if they only changed the SDK connection name, that has no impact on the content returned, so we can skip the purge.